src/go/rdctl/pkg/factoryreset/factory_reset_windows.go

```diff
@@ -30,10 +31,48 @@ import (
 	"golang.org/x/text/encoding/unicode"
 )
 
-func KillRancherDesktop() {
+func KillRancherDesktop() error {
 	cmd := exec.Command("taskkill", "/IM", "Rancher Desktop.exe", "/T", "/F")
 	cmd.SysProcAttr = &syscall.SysProcAttr{CreationFlags: CREATE_NO_WINDOW}
-	cmd.Run()
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("Failed to taskkill Rancher Desktop.exe: %w", err)
+	}
+	return nil
+}
+
+//WARNING: This will fail if we localize the name of the app
+
+func CheckProcessWindows() (bool, error) {
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+
+	cmd := exec.Command("tasklist", "/FI", "WINDOWTITLE eq Rancher Desktop", "/FO", "List")
```

Comment:
Isn't `/FI "IMAGENAME eq Rancher Desktop.exe"` (like how you had for `taskkill`) better?

In particular, this is wrong if the user closed the main window (i.e. it's running in the tray).

Reply:
Good catch, thx